api/src/test/java/org/pmiops/workbench/utils/mappers/WorkspaceMapperTest.java

```diff
@@ -171,6 +174,18 @@ public void testConvertsDbToApiWorkspace() {
 
     assertThat(ws.getCreationTime()).isEqualTo(DB_CREATION_TIMESTAMP.toInstant().toEpochMilli());
     assertThat(ws.isPublished()).isEqualTo(sourceDbWorkspace.getPublished());
+    assertThat(ws.getFeaturedCategory()).isNull();
```

Comment:
this confirms that it's safe to remove the `isFeaturedWorkspace()` check from WorkspaceMapper